### PR TITLE
Phase 3 Funding: Infrastructure and core/coordinated package maintenance (Aperio Software)

### PR DESCRIPTION
**Pull request by astrofrog**
This is an initial draft of a Phase 3 proposal that covers:

* Maintenance of project infrastructure (such as pytest and sphinx plugins,  extension-helpers, continuous integration, wheel building, and so on).
* Maintenance and development of the  [astropy.timeseries](https://docs.astropy.org/en/stable/timeseries/) and  [astropy.visualization.wcsaxes](https://docs.astropy.org/en/stable/visualization/wcsaxes/)  sub-packages as well as the [reproject](http://reproject.readthedocs.io) and   [astropy-healpix](https://astropy-healpix.readthedocs.io/en/latest/)  coordinated packages.

### CODE REVIEW

**Comment by kelle** [comment]
Please react to this comment to vote on this proposal (👍, 👎, or no reaction for +0).

File: finance/proposal-calls/cycle3/Aperio-Infrastructure-Maintenance.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+  [astropy.visualization.wcsaxes](https://docs.astropy.org/en/stable/visualization/wcsaxes/)
+  sub-packages as well as the [reproject](http://reproject.readthedocs.io) and
+  [astropy-healpix](https://astropy-healpix.readthedocs.io/en/latest/)
+  coordinated packages.
+
+### Project / work
+
+#### Infrastructure work
+
+We will participate in the maintenance of the existing project infrastructure,
```
Comment (pllim) [inline finance/proposal-calls/cycle3/Aperio-Infrastructure-Maintenance.md:35]: While we can really use the help, the infrastructure side continues to struggle (i.e., issues over at pytest and doctest plugins continue to linger on). I wonder if Aperio is getting stretched too thin across too many things. I see you have proposed multiple stuff as separate issues.

Comment (Cadair) [reply finance/proposal-calls/cycle3/Aperio-Infrastructure-Maintenance.md:35]: Speaking just for me, this funding request is really for us to continue doing what I can, as I have been. Other things I am working on (mainly not Astropy work) have to take priority most of the time.

**Comment by jdswinbank** [comment]
Congratulations!  The community has expressed support for this FR, and the Finance Committee has determined it can be funded.  This FR has been approved for $86,400, to be funded from the Moore grant.

@eteq is the Finance Committee member who will be supporting this project.  They will update this PR with the amount and then merge it, as well as being in contact with you to set up any necessary agreements and relevant Open Collective financial arrangements. They will also open a new issue where progress on this project can be discussed.

**Pull request merged**